### PR TITLE
Add synchronizer cji to deployment

### PR DESCRIPTION
**Pull request by fijshion**
# Overview

This PR is being created to address [ESSNTL-xxxx](https://issues.redhat.com/browse/ESSNTL-xxxx).
(A description of your PR's changes, along with why/context to the PR, goes here.)

## PR Checklist

- [ ] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [ ] Recovers or fails gracefully during potential resource outages (e.g. DB, Kafka)
- [ ] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Documentation Reference

You can find documentation on this checklist [here](https://github.com/RedHatInsights/secure-coding-checklist).

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: deployment.yaml [hunk 1]
```diff
@@ -782,6 +790,9 @@ parameters:
 - name: CJI_RANDOM_SUFFIX
   required: true
   value: '123456'
+- name: SYNCHRONIZER_CJI_RANDOM_SUFFIX
```
Comment (thearifismail) [inline deployment.yaml:793]: IMO, `CJI_RANDOM_SUFFIX` should be enough for all CJIs

Comment (fijshion) [reply deployment.yaml:793]: if we have a single CJI_RANDOM_SUFFIX, then each time it is changed, all jobs will run.

Comment (thearifismail) [reply deployment.yaml:793]: that's what I thought should be done but we may not want to do that.

**Review by thearifismail — APPROVED** [review]
LGTM

**Review by ryandillinfelton — APPROVED** [review]

**Pull request merged**